database/factories/MessageFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Message;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+
+class MessageFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Message::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'user_id' => User::all()->random(),
+            'content' => Str::random(60),
```

Comment:
(suggest)
こっちのほうがよりそれっぽい文章が生成されるかと思います。

```suggestion
            'content' => $this->faker->realText(60),
```